yellow-paper/docs/public-vm/gen/_InstructionSet.mdx

```diff
@@ -0,0 +1,989 @@
+- **Args**: 
+	- **dstOffset**: memory offset specifying where to store operation's result
+- **Expression**: `M[dstOffset] = TxContext.feePerL1Gas`
+- **Tag suboperations**: `T[dstOffset] = u32`
+
+
+### `FEEPERL2GAS` <a id='isa-section-feeperl2gas'/>
+The fee to be paid per "L2 gas" - set by the transaction's original caller
+
+[See in table.](#isa-table-feeperl2gas)
+
+- **Category**: tx context
+- **Args**: 
+	- **dstOffset**: memory offset specifying where to store operation's result
+- **Expression**: `M[dstOffset] = TxContext.feePerL2Gas`
+- **Tag suboperations**: `T[dstOffset] = u32`
+
+
+### `CALLER` <a id='isa-section-caller'/>
+Get the address of the sender (the caller's context)
+
+[See in table.](#isa-table-caller)
+
+- **Category**: call context
+- **Args**: 
+	- **dstOffset**: memory offset specifying where to store operation's result
+- **Expression**: `M[dstOffset] = CallContext.sender`
+- **Tag suboperations**: `T[dstOffset] = u32`
+
+
+### `ADDRESS` <a id='isa-section-address'/>
+Get the address of the currently executing l2 contract
+
+[See in table.](#isa-table-address)
+
+- **Category**: call context
+- **Args**: 
+	- **dstOffset**: memory offset specifying where to store operation's result
+- **Expression**: `M[dstOffset] = CallContext.storageContractAddress`
+- **Tag suboperations**: `T[dstOffset] = u32`
+
+
+### `PORTAL` <a id='isa-section-portal'/>
+Get the address of the l1 portal contract
+
+[See in table.](#isa-table-portal)
+
+- **Category**: call context
+- **Args**: 
+	- **dstOffset**: memory offset specifying where to store operation's result
+- **Expression**: `M[dstOffset] = CallContext.portalAddress`
+- **Tag suboperations**: `T[dstOffset] = u32`
+
+
+### `CALLDEPTH` <a id='isa-section-calldepth'/>
+Get how many calls deep the current call context is
+
+[See in table.](#isa-table-calldepth)
+
+- **Category**: call context
+- **Args**: 
+	- **dstOffset**: memory offset specifying where to store operation's result
+- **Expression**: `M[dstOffset] = CallContext.calldepth`
+- **Details**: Note: security issues with EVM's tx.origin can be resolved by asserting the `calldepth == 0`.
+- **Tag suboperations**: `T[dstOffset] = u8`
+
+
+### `L1GAS` <a id='isa-section-l1gas'/>
+Remaining "L1 gas" for this call (after this instruction).
+
+[See in table.](#isa-table-l1gas)
+
+- **Category**: latest context
+- **Args**: 
+	- **dstOffset**: memory offset specifying where to store operation's result
+- **Expression**: `M[dstOffset] = LatestContext.l1Gas`
+- **Tag suboperations**: `T[dstOffset] = u32`
+
+
+### `L2GAS` <a id='isa-section-l2gas'/>
+Remaining "L2 gas" for this call (after this instruction).
+
+[See in table.](#isa-table-l2gas)
+
+- **Category**: latest context
+- **Args**: 
+	- **dstOffset**: memory offset specifying where to store operation's result
+- **Expression**: `M[dstOffset] = LatestContext.l2Gas`
+- **Tag suboperations**: `T[dstOffset] = u32`
+
+
+### `<HISTORICROOT>` <a id='isa-section-historicroot'/>
```

Comment:
@dbanks12 This section title lies between angle brackets <>. Is this on purpose or a leftover?

Reply:
Removed them